core-java-modules/core-java-lang-4/src/main/java/com/baeldung/bookstore/Book.java

```diff
@@ -0,0 +1,17 @@
+//incorrect package com.baeldung declaration;
+package com.baeldung.bookstore;
+
+public class Book {
+
+	private String title;
```

Comment:
These files are formatted with tabs, where it should be spaces.

Reply:
I have reformatted using 4 spaces.